doc/features/mapper/queries/README.md

```diff
@@ -0,0 +1,294 @@
+# Queries
+
+The Mapper generates the CQL queries and adapts the results into objects according to the [defined
+mappings](../defining-mappings/).
+
+*Note that throughout the Mapper documentation the [killrvideo schema][killrvideo] is used.* 
+
+## Retrieval
+
+The are three methods in the `ModelMapper` that are used to retrieve objects from the database:
+
+- `find()`: filters by one or more primary keys and returns the `Result` that is an iterable of objects.
+- `get()`: Gets one document matching the provided filter or `null` when not found. Note that all partition and 
+clustering keys must be defined in order to use this method.
+- `findAll()`: selects all the objects returns the `Result` that is an iterable of objects. This is only recommended
```

Comment:
Do we want to eventually offer a solution for paging, similar to `stream()` or `eachRow()`?  I think that would be nice to have, although maybe don't need it for first iteration.  I opened [NODEJS-480](https://datastax-oss.atlassian.net/browse/NODEJS-480) to pursue this further.